apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/user-bookings.e2e-spec.ts

```diff
@@ -1936,292 +1939,468 @@ describe("Bookings Endpoints 2024-08-13", () => {
-          start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
-          eventTypeId: eventTypeWithAllLocationsId,
-          attendee: {
-            name: "Mr Proper",
-            email: "mr_proper@gmail.com",
-            timeZone: "Europe/Rome",
-            language: "it",
-          },
-          location: {
-            type: "attendeeAddress",
-            address: attendeeAddress,
-          },
-        };
+        it("can't book with not location that is not in event type", async () => {
+          const bookingBody: CreateBookingInput_2024_08_13 = {
+            start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
+            eventTypeId: eventTypeWithAllLocationsId,
+            attendee: {
+              name: "Mr Proper",
+              email: "mr_proper@gmail.com",
+              timeZone: "Europe/Rome",
+              language: "it",
+            },
+            location: {
+              type: "integration",
+              integration: "google-meet",
+            },
+          };
 
-        const bookingResponse = await request(app.getHttpServer())
-          .post("/v2/bookings")
-          .send(bookingBody)
-          .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
-          .expect(201);
+          await request(app.getHttpServer())
+            .post("/v2/bookings")
+            .send(bookingBody)
+            .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
+            .expect(400);
+        });
 
-        const bookingResponseBody: CreateBookingOutput_2024_08_13 = bookingResponse.body;
-        const createdBooking = bookingResponseBody.data;
-        expect(createdBooking).toHaveProperty("id");
+        it("can't book with invalid location type", async () => {
+          const bookingBody = {
+            start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
+            eventTypeId: eventTypeWithAllLocationsId,
+            attendee: {
+              name: "Mr Proper",
+              email: "mr_proper@gmail.com",
+              timeZone: "Europe/Rome",
+              language: "it",
+            },
+            location: {
+              type: "blablabala",
+            },
+          };
 
-        if (responseDataIsBooking(createdBooking)) {
-          expect(createdBooking.location).toEqual(attendeeAddress);
-          await bookingsRepositoryFixture.deleteById(createdBooking.id);
-        } else {
-          throw new Error("Unexpected response data type");
-        }
+          await request(app.getHttpServer())
+            .post("/v2/bookings")
+            .send(bookingBody)
+            .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
+            .expect(400);
+        });
       });
 
-      xit("can book with attendeeAddress location", async () => {
-        const attendeePhone = "+37120993151";
+      // then add one more test below
+      // describe("can update booking with multiple locations")
+      describe("Can update booking with multiple locations", async () => {
```

Comment:
I would rename it "can update with different locations" because you can book with only 1 location